pages/docs/castor/rao-steps.md

```diff
@@ -40,11 +40,34 @@ Overloads potentially reached on this perimeter are to be solved with preventive
 
 ## Automaton perimeters {#auto-rao}
 
-An automaton perimeter can be defined for each defined contingency. Each automaton perimeter is formed by:
-- CNECs of the “after automatons” defined with a limit of type TATL. All CNECs are defined upon the same contingency.
+An automaton perimeter can be defined for each defined contingency. Each automaton perimeter is formed by CNECs of the “after automatons” defined with a limit of type TATL. All CNECs are defined upon the same contingency.
 
-Overloads potentially reached on this perimeter can possibly be solved with automatons (ARA). 
-Optimal preventive actions selected by the preventive RAO are applied in this perimeter.  
+Optimal preventive actions selected by the preventive RAO are applied in this perimeter.   
+
+Preventive and curative perimeters optimized overloads on their respective perimeters by applying available remedial actions, whereas the automaton perimeter performs a **simulation**
```

Comment:
This sentence is difficult to read.
I find it easier this way:
"While preventive and curative remedial actions are **optimized** to prevent overloads in their respective perimeters, auto remedial actions are actually **simulated** in the automaton perimeter."